x/staking/simulation/operations_test.go

```diff
@@ -167,7 +168,7 @@ func (s *SimTestSuite) TestWeightedOperations() {
 func (s *SimTestSuite) TestSimulateMsgCreateValidator() {
 	require := s.Require()
 	// begin a new block
```

Comment:
can we update the comment, 

i can see simulations being a pain in the near future cause the flow is all different

Reply:
I think we should revisit those comments, I can see lots of such cases